### PR TITLE
Fix #283 & #321 path issues [+1 for Windows]

### LINKED ISSUE #283: Error when for 0.4.4. Refused to execute script from "*.js" because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
Hi Guys, was working good with 0.3.3. but got bellow error with 0.4.0.

Refused to execute script from 'http://localhost:3000/modules/core/controllers/header.client.controller.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
meanjs0 4 error

I am using window 8.1. node js v0.10.24.

### PR DESCRIPTION
**Pull request by AndreasHassing**
This fixes https://groups.google.com/forum/#!msg/meanjs/qOPHzMtWrPY/da9puKbyI5UJ as suggested by Silla Tan - also mentioned in #283 which was closed without a fix.

### CODE REVIEW

**Comment by AndreasHassing** [comment]
Works on OSX, however I realized that the {{url}} field generates some pretty weird URLs, for instance `http://localhost:3000/authentication/signinmodules/core/img/brand/logo.png`. Working on it.

File: modules/core/server/views/layout.server.view.html [hunk 1]
```diff
@@ -26,14 +26,14 @@
...
-	<meta property="og:image" content="/img/brand/logo.png">
+	<meta property="og:image" content="{{url}}modules/core/img/brand/logo.png">
 	<meta property="og:type" content="website">
 
 	<!-- Twitter META -->
 	<meta name="twitter:title" content="{{title}}">
 	<meta name="twitter:description" content="{{description}}">
 	<meta name="twitter:url" content="{{url}}">
-	<meta name="twitter:image" content="/img/brand/logo.png">
+	<meta name="twitter:image" content="{{url}}modules/core/img/brand/logo.png">
```
Comment (ilanbiala) [inline modules/core/server/views/layout.server.view.html]: Shouldn't it just be `/modules/core/img/brand/logo.png`?

Comment (AndreasHassing) [inline modules/core/server/views/layout.server.view.html]: @ilanbiala You need to full URL to satisfy twitter and facebook. See @simison's comment at https://github.com/meanjs/mean/pull/322

**Comment by NeverOddOrEven** [comment]
@AndreasHassing or @ilanbiala please create a new issue for the view template.

**Pull request merged**